### ISSUE TITLE
Escaped dollars inside equations are not working.

### ISSUE THREAD

**Issue by garciparedes** [vberlier/extract-math#1]
I write inline equations like `$45\$$` but I can't escape the escape the dollar symbol inside the equation. The result of this is that equation is ignored.

**Comment by vberlier** [comment]
```js
extractMath('$45\\$$') // [{math: true, type: 'inline', value: '45$'}]
```

I think the behavior is correct. The function extracts the expression and replaces the escaped dollars. However it does seem like KaTeX doesn't handle bare dollar signs in expressions:

![image](https://user-images.githubusercontent.com/24430071/55292001-6299aa80-53b3-11e9-91be-a2b7cbabaf79.png)

A solution would be to add an `escaped` property to the returned segments containing a string that can be passed directly to KaTeX:

```js
extractMath('$45\\$$') // [{math: true, type: 'inline', value: '45$', escaped: '45\\$'}]
```

![image](https://user-images.githubusercontent.com/24430071/55292098-785b9f80-53b4-11e9-836e-b6deb2f9f000.png)

**Comment by vberlier** [comment]
Actually, I think an even better solution would be to add a property exposing the raw value instead. This way you have access to the original string and it lets you use it in situations where it's preferable to not interpret escape sequences, like here with KaTeX.

**Comment by vberlier** [comment]
Just released 1.0.3 which introduces a `raw` property e85cd2bc19f8501d7c2e635f05727610bc965cb5.
